datalad/distributed/create_sibling_ria.py

```diff
@@ -52,9 +56,11 @@
     decode_source_spec
 )
 from datalad.log import log_progress
-from datalad.distributed.ria_utils import (
+from datalad.customremotes.ria_utils import (
```

Comment:
I don't like this move. We should stick to the plan to slowly move code into the new package structure. https://github.com/datalad/datalad/issues/3192

Reply:
Yeah, I'm not entirely clear about how to deal with the actual special remote. #3192 is mostly concerned with "commands". In #4164 you moved parts into `customremotes` and parts into 
`distributed`. I had the impression that the actual special remote should be in `customremotes` while related commands are supposed to go into `distributed`.

Reply:
I can live with that. Thx for the explanation.